AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/hydra_SubID_NoChange_MeshChanged.py

```diff
@@ -0,0 +1,78 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+class Tests:
+    asset_id_no_change = (
+        "Model AssetId matches after changes",
+        "P0: Model AssetId does not match expected output")
+
+def SubID_NoChange_MeshChanged():
+
+    import os
+    import shutil
+    import time
+    from pathlib import Path
+
+    from Atom.atom_utils.atom_constants import AtomComponentProperties
+    from EditorPythonTestTools.editor_python_test_tools.editor_entity_utils import EditorEntity
+    from EditorPythonTestTools.editor_python_test_tools.utils import Report, TestHelper, Tracer
+
+    with Tracer() as error_tracer:
+        # Test Setup: Wait for Editor idle before loading the level and executing Python hydra scripts.
+        TestHelper.init_idle()
+        TestHelper.open_level("AssetPipeline", "SceneTests")
+
+        # Test Setup: Set up source and destination for assetinfo file, then verify there is no file there already.
+        dirpath = Path(__file__).parents[4]
+        src = os.path.join(dirpath, 'Objects', 'ShaderBall_simple', 'shaderball_simple_MeshChange_SameID.fbx.assetinfo')
+        dst = os.path.join(dirpath, 'Objects', 'shaderball_simple.fbx.assetinfo')
+        if os.path.exists(dst):
+            os.remove(dst)
+            time.sleep(5) # Allow enough time for AP to pick up and process the test asset, replace with APIdle check once bus is exposed to python bindings.
```

Comment:
Is there a way you can check the output instead of having this sleep? These sleeps are going to be frail, especially if these tests ever run in parallel. No problem if not, then we can cut a ticket to the backlog to investigate later.